packages/deployments/contracts/contracts/Connext.sol

```diff
@@ -140,6 +140,11 @@ contract Connext is
    */
   mapping(address => mapping(address => uint256)) public routerBalances;
 
+  /**
+   * @notice The max amount of routers a payment can be routed through
+   */
+  uint256 public maxRouters;
```

Comment:
can we change this to `multipathMaxRouters` ?

Reply:
Just concerned it might be misconstrued as "max routers allowed in the network for this chain" or something lol

Reply:
Additionally, would need to change event names below to follow that lol

Reply:
will change it to `maxRoutersPerTransfer`